qa/pull-tester/run-zcashd-for-test.sh.in

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-# Copyright (c) 2013-2014 The Bitcoin Core developers
```

Comment:
This is an incorrect change. It's fine to add copyright notices for "The Zcash Core developers", but notices for "The Bitcoin Core developers" should be retained in existing files or derived works of existing files.